docs/hub/billing.md

```diff
@@ -1,30 +1,70 @@
 
-We also partner with cloud providers, like [AWS](https://huggingface.co/blog/aws-partnership) and [Azure](https://huggingface.co/blog/hugging-face-endpoints-on-azure), to make it easy for customers to use Hugging Face directly in their cloud of choice. These solutions and usage are billed directly by the cloud provider. Ultimately we want people to be able to have great options to use Hugging Face wherever they build Machine Learning.
 
-All user and organization accounts have a billing system. You can submit your payment info to access "pay-as-you-go" services.
+## Cloud providers partnerships
 
-From the [Settings > Billing](https://huggingface.co/settings/billing) page, you can see a real time view of your paid usage across all HF services, for instance:
+We partner with cloud providers, like [AWS](https://huggingface.co/blog/aws-partnership) and [Azure](https://huggingface.co/blog/hugging-face-endpoints-on-azure), to make it easy for customers to use Hugging Face directly in their cloud of choice. These solutions and usage are billed directly by the cloud provider. Ultimately we want people to be able to have great options to use Hugging Face wherever they build Machine Learning.
 
-- [Spaces](./spaces),
-- [AutoTrain](https://huggingface.co/docs/autotrain/index),
-- [Inference Endpoints](https://huggingface.co/docs/inference-endpoints/index)
-- [PRO subscription](https://huggingface.co/pricing) (for users)
-- [Enterprise Hub subscription](https://huggingface.co/enterprise) (for organizations)
+## Compute Services on the Hub
+
+We also directly provide compute services with [Spaces](./spaces), [Inference Endpoints](https://huggingface.co/docs/inference-endpoints/index) and the [Serverless Inference API](https://huggingface.co/docs/api-inference/index).
+
+While most of our compute services have a comprehensive free-tier, users and organizations can pay to access more powerful hardware accelerators.
+
+The billing for our compute services is usage-based, meaning you only pay for what you use. You can monitor your usage at any time from your billing dashboard, located in the settings menu of your user or organization.
 
 <div class="flex justify-center">
-<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/billing.png"/>
-<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/billing-dark.png"/>
+	<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/refs%2Fpr%2F357/hub/billing/billing-dashboard-light.png"/>
+	<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/refs%2Fpr%2F357/hub/billing/billing-dashboard-dark.png"/>
 </div>
 
-Any feedback or support request related to billing is welcome at billing@huggingface.co.
```

Comment:
Do we want to keep this line?

Reply:
I can re-add it if you think it is impactful